src/machine learning analysis.py

```diff
@@ -0,0 +1,155 @@
+# author: Siqi Tao
```

Comment:
Let us call the script `model.py` to tie back to the flowchart

Reply:
done